gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 #Thu Apr 29 21:07:26 PDT 2021
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-bin.zip
```

Comment:
IMO `all` is better than `bin` so you get sources / documentation available in IDE

Reply:
When writing Gradle plugins or using more complicated build logic I use the `all` distribution, but for simple stuff like this it's not really necessary imo, and just ends up causing people to download more Gradle distributions when trying to build the mod.